### PR TITLE
Fixes another instance of fortnite gaming on SS13

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
fix: You can no longer fortnite with glass sheets
/:cl:

### CODE REVIEW

**Comment by TheVekter** [comment]
Can we get the FUCK tag please

File: code/game/objects/items/stacks/sheets/glass.dm [hunk 1]
```diff
@@ -9,8 +9,8 @@
  * Glass sheets
  */
 GLOBAL_LIST_INIT(glass_recipes, list ( \
-	new/datum/stack_recipe("directional window", /obj/structure/window/unanchored, time = 0, on_solid_ground = TRUE, window_checks = TRUE, category = CAT_WINDOWS), \
-	new/datum/stack_recipe("fulltile window", /obj/structure/window/fulltile/unanchored, 2, time = 0, on_solid_ground = TRUE, window_checks = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("directional window", /obj/structure/window/unanchored, time = 5 SECONDS, on_solid_ground = TRUE, window_checks = TRUE, category = CAT_WINDOWS), \
+	new/datum/stack_recipe("fulltile window", /obj/structure/window/fulltile/unanchored, 2, time =  5 SECONDS, on_solid_ground = TRUE, window_checks = TRUE, category = CAT_WINDOWS), \
```
Comment (MrStonedOne) [inline code/game/objects/items/stacks/sheets/glass.dm:13]:
```suggestion
	new/datum/stack_recipe("directional window", /obj/structure/window/unanchored, time = 0.5 SECONDS, on_solid_ground = TRUE, window_checks = TRUE, category = CAT_WINDOWS), \
	new/datum/stack_recipe("fulltile window", /obj/structure/window/fulltile/unanchored, 2, time =  1 SECONDS, on_solid_ground = TRUE, window_checks = TRUE, category = CAT_WINDOWS), \
```

If you start smaller you'll have an easier time getting this approved.

Comment (tralezab) [reply code/game/objects/items/stacks/sheets/glass.dm:13]: Great suggestion

**Comment by Pickle-Coding** [comment]
What about the other glasses recipes? Looks lile there is a lot of copy pasting to do from my understanding.

**Comment by HansRobertCoooding** [comment]
Yeah we got number one victory royale...

File: code/game/objects/items/stacks/sheets/glass.dm (hunk 1 above)
Comment (Kylerace) [reply code/game/objects/items/stacks/sheets/glass.dm:13]: 0.5 or 1 second is fine

**Comment by Jackal-boop** [comment]
this will greatly effect the stations chances during war ops

**Comment by moocowswag** [comment]
https://www.youtube.com/watch?v=umALQ4MTgz0

I posted this on discord, it took goof less than 5 minutes to make this pr, and the video is 1 minute long so less than 4 minutes..

Maybe spend more time on the pr and test it.

**Comment by ArthuroPep** [comment]
what the FUCK does this even mean

**Comment by flowercuco** [comment]
goofball please explain how you are changing the game and what the problem is in these prs

**Review by Mothblocks — CHANGES_REQUESTED** [review]
please fix your title, description, and changelog to be understandable to people over the age of 15

**Review by Fikou — CHANGES_REQUESTED** [review]
you forgot every other glass type

**Comment by DAKKA-WAAAGH** [comment]
Fork knife

**Comment by NamelessFairy** [comment]
Not really a huge issue but just want to bring up the idea. Would it be possible to shorten the construction step process of windows by the same amount so the overall window building time is still the same as before this PR if your just building them for construction purposes.

**Comment by UnokiAs** [comment]
What does this pr do ?
Increase window build time ?

**Comment by JohnFulpWillard** [comment]
> please fix your title, description, and changelog to be understandable to people over the age of 15

https://www.youtube.com/shorts/Bl1ESD1YS4I

**Comment by optimumtact** [comment]
mfw maintainers don't play fortnite

**Pull request merged**